README.md

```diff
@@ -17,6 +17,10 @@ or use our bot: https://telegram.me/CoredumpBot
     cd coredumpbot
     TELEGRAM_BOT_TOKEN=XXXXXXXXXXXXXXXXXXXXXXX cargo run --release
 
+# Install service file
+
+    sudo ln -s $HOME/coredumpbot.service /etc/systemd/system/multi-user.target.wants/coredumpbot.service
```

Comment:
Why $HOME?

Also, you shouldn't symlinks service files manually like this. Instead, add them to /etc/systemd/system/ and use `systemctl enable <service>` instead.